src/main/java/com/fasterxml/jackson/databind/deser/impl/PropertyValueBuffer.java

```diff
@@ -179,6 +181,32 @@ public Object[] getParameters(SettableBeanProperty[] props)
         return _creatorParameters;
     }
 
+    public Object[] getParametersWithAnySetter(SettableBeanProperty[] props, SettableAnyProperty anySetter)
+        throws JsonMappingException
+    {
+        getParameters(props);
```

Comment:
Should probably make use of return value: I know in practice it will be `_creatorParameters` but probably should not assume that. So something like

```
Object[] params = getParameters(props);
```